manifests/m/Microsoft/PowerShell/Preview/7.4.0.2/Microsoft.PowerShell.Preview.installer.yaml

```
@@ -0,0 +1,45 @@
+# Created with YamlCreate.ps1 v2.2.3 $debug=NVS1.CRLF.7-3-3.Win32NT
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.4.0.schema.json
+
+PackageIdentifier: Microsoft.PowerShell.Preview
+PackageVersion: 7.4.0.2
+MinimumOSVersion: 10.0.17763.0
+InstallerType: wix
+Scope: machine
+InstallModes:
+- interactive
+- silent
+- silentWithProgress
+UpgradeBehavior: install
+Commands:
+- pwsh
+ReleaseDate: 2023-03-14
+Installers:
+- Architecture: x64
+  InstallerUrl: https://github.com/PowerShell/PowerShell/releases/download/v7.4.0-preview.2/PowerShell-7.4.0-preview.2-win-x64.msi
+  InstallerSha256: 781AE74C9A9403409D577C1CCD699631A406DCB104B9023991731BCE60F128C3
+  ProductCode: '{33E0910E-D1FB-4FB8-A72B-DF212502E4CF}'
+  AppsAndFeaturesEntries:
+  - Publisher: Microsoft Corporation
+    DisplayName: PowerShell 7-preview-x64
+    DisplayVersion: 7.4.0.2
+- Architecture: x86
+  InstallerUrl: https://github.com/PowerShell/PowerShell/releases/download/v7.4.0-preview.2/PowerShell-7.4.0-preview.2-win-x86.msi
+  InstallerSha256: CF2DA74890738FA9845FC201FC1E657EF4C4121D2FEDBE78B9DA09B041FEF69E
+  ProductCode: '{095D635E-5CDB-4FD1-AF2C-A07CC2F83DB5}'
+  AppsAndFeaturesEntries:
+  - Publisher: Microsoft Corporation
+    DisplayName: PowerShell 7-preview-x86
+    DisplayVersion: 7.4.0.2
+- Architecture: arm
+  InstallerType: msix
+  InstallerUrl: https://github.com/PowerShell/PowerShell/releases/download/v7.4.0-preview.2/PowerShell-7.4.0-preview.2-win.msixbundle
+  InstallerSha256: 6B3B431DFDDD708297507CF08885A59CEE327DAEF9CC7064823B2D582FBED720
+  SignatureSha256: 04C3667D7AF40D9B2F6CC21181EF1D9BF5DF2A8F1C5706FF5E4753A70FFD61E5
```

```suggestion
  SignatureSha256: 04C3667D7AF40D9B2F6CC21181EF1D9BF5DF2A8F1C5706FF5E4753A70FFD61E5
  PackageFamilyName: Microsoft.PowerShellPreview_8wekyb3d8bbwe
  AppsAndFeaturesEntries:
  - Publisher: Microsoft Corporation
    DisplayVersion: 7.4.2.0
```
Since you changed the PackageVersion, we're gonna need DisplayVersion for the msix installers now.